src/main/java/org/openpnp/gui/tablemodel/BoardLocationsTableModel.java

```diff
@@ -51,11 +57,18 @@ public void setJob(Job job) {
         this.job = job;
         fireTableDataChanged();
     }
-
+    
     public Job getJob() {
         return job;
     }
 
+    public BoardStatus getBoardStatus(int index) {
+    	if (Double.compare(job.getBoardLocations().get(index).getLocation().getLengthZ().getValue(), 0.0) == 0) {
```

Comment:
This can be simplified to (job.getBoardLocations().get(index).getZ() == 0), no need for the complex compare. Not a critical change though.

Reply:
done